lib/index.js

```diff
@@ -0,0 +1,3 @@
+import Dredd from './Dredd';
+
+module.exports = Dredd;
```

Comment:
I'm adding explicit entry module so it can be exported properly.

By default, TypeScript will compile the source code into CJS, but preserve the default/named exports. This means that if you wish to import a distributed Dredd you cannot do:

```js
const Dredd = require('dredd')
```

You would have to do:

```js
const Dredd = require('dredd').default
```

To solve this, an explicit entry helps. It exports the default export as `module.exports`, allowing to preserve the same require/import as before.

Reply:
This should probably be a comment in code.

Reply:
Added in 2f1a89d.